README.md

```diff
@@ -94,10 +96,23 @@ $ ./do delete:docker      # stop and remove all running docker containers.
 $ ./do qa:lint             # PHP code linter.
 $ ./do qa:lint:javascript  # JS code linter.
 $ ./do qa                  # PHP and JS linters.
+
+$ ./do container:dump      # Dumps DI container.
```

Comment:
This is not very helpful. The second part `# Dumps DI container.` doesn't say anything above the first part and doesn't tell us why should we dump the container.